internal/runners/hello/info_example.go

```diff
@@ -0,0 +1,135 @@
+package hello
```

Comment:
This runner code is too opinionated. The point of the example is to show basic standards. I think a lot of what is happening here isn't standards so much as how you individually would approach the problem. Which is totally fair for an individual runner you authored, but not something I want to portray as a standard.

Anyway, we don't need two example runners. One is enough. Suggest getting rid of this one.